tests/test.js

```diff
@@ -13,7 +13,11 @@ describe('Promise Matcher tests', function () {
   beforeEach(function() {
     angular.mock.module('foobar');
 
-    installPromiseMatchers();
+    installPromiseMatchers({
+      httpBackend: true,
+      interval: true,
+      timeout: true
+    });
```

Comment:
Would be nice to have a test that verifies the newly added behavior (e.g. that flush can be disabled)